git-link.el

```diff
@@ -103,18 +103,28 @@
   "If non-nil use the latest commit's hash in the link instead of the branch name.")
 
 (defvar git-link-remote-alist
-  '(("github.com"    git-link-github)
-    ("bitbucket.org" git-link-bitbucket)
-    ("gitorious.org" git-link-gitorious)
-    ("gitlab.com"    git-link-gitlab))
-  "Maps remote hostnames to a function capable of creating the appropriate file URL")
+  '(("github" git-link-github)
+    ("bitbucket" git-link-bitbucket)
+    ("gitorious" git-link-gitorious)
+    ("gitlab" git-link-gitlab))
+  "Alist containing (REGEXP FUNCTION) where REGEXP is used to
+match the remote host URL and FUNCTION is capable of creating the
+appropriate file URL.
+
+As an example, \"gitlab\" will match with both \"gitlab.com\" and
+\"gitlab.COMPANY.com\".")
 
 (defvar git-link-commit-remote-alist
-  '(("github.com"    git-link-commit-github)
-    ("bitbucket.org" git-link-commit-bitbucket)
-    ("gitorious.org" git-link-commit-gitorious)
-    ("gitlab.com"    git-link-commit-github))
-  "Maps remote hostnames to a function capable of creating the appropriate commit URL")
+  '(("github" git-link-commit-github)
+    ("bitbucket" git-link-commit-bitbucket)
+    ("gitorious" git-link-commit-gitorious)
+    ("gitlab" git-link-commit-github))
+  "Alist containing (REGEXP FUNCTION) where REGEXP is used to
+match the remote host URL and FUNCTION is capable of creating the
+appropriate commit URL.
+
+As an example, \"gitlab\" will match with both \"gitlab.com\" and
+\"gitlab.COMPANY.com\".")
```

Comment:
Since doc strings use capitals for arguments should "COMPANY"  really be capitalized here?